src/OrchardCore/OrchardCore.ContentManagement.GraphQL/Queries/Types/DynamicContentTypeBuilder.cs

```diff
@@ -12,6 +12,11 @@ namespace OrchardCore.ContentManagement.GraphQL.Queries.Types
 {
     public class DynamicContentTypeBuilder : IContentTypeBuilder
     {
+        // If registered, then fields input should be exposed, otherwise ignored.
+        public class FieldsInputMarker
+        {
+        }
```

Comment:
This type appears to be used only to represent a function switch, and it is recommended to use IOptions, e.g., by adding a property to `GraphQLContentOptions`. : )

Reply:
Done, I removed this marker by creating different builders for query and input fields